hawc_hal/maptree/__init__.py

```diff
@@ -1 +1,3 @@
-from map_tree import map_tree_factory
\ No newline at end of file
+from map_tree import map_tree_factory
+from map_tree import MapTree
```

Comment:
What is the need of adding these here?

Adding an import on a `__init__` means that the user will have access to those classes. Since the user should not create a `MapTree` directly, nor a `DataAnalysisBin` directly (these are for internal use), we should not import them into this `__init__`.

If you need to have access to those from another subdirectory, you can just say:

`from hawc_hal.maptree.map_tree import MapTree`

for example.